types/node/module.d.ts

```diff
@@ -1,3 +1,19 @@
 declare module "module" {
-    export = NodeJS.Module;
+    import { URL } from "url";
+    interface Module extends NodeJS.Module {}
```

Comment:
Seems the move of `Module` causes problems in one of my applications.
I augment `Module` with internals but I'm not able to adapt it in a way compatible to your changes:
```
declare namespace NodeJS {
  namespace Module {
    function _load(request: string, parent: NodeModule | undefined, isMain?: boolean): any;
    function _resolveFilename(request: string, parent: NodeModule | undefined): string;
    var _cache: ModuleCache;
  }
}
```

Reply:
I'm able to adapt my code by moving the augmentation into `module "module"` and change from `import Module = require("module");` to `import { Module } from "module";`
```
declare module "module" {
  namespace Module {
    function _load(request: string, parent: NodeModule | undefined, isMain?: boolean): any;
    function _resolveFilename(request: string, parent: NodeModule | undefined): string;
    var _cache: NodeJS.ModuleCache;
  }
} 
```